keps/sig-release/3720-freezing-k8s-gcr-io/kep.yaml

```diff
@@ -0,0 +1,23 @@
+title: Freezing `k8s.gcr.io` image registry
+kep-number: 3720
+authors:
+  - "@upodroid"
+owning-sig: sig-release
+participating-sigs:
+  - sig-k8s-infra
+  - sig-release
+status: implementable
+creation-date: 2023-01-10
+reviewers:
+  - "@BenTheElder"
+  - "@puerco"
+  - "@ameukam"
```

Comment:
```suggestion
reviewers:
  - "@ameukam"
  - "@dims"
  - "@jeremyrickard"
  - "@justaugustus"
  - "@saschagrunert"
```

Reply:
Yeah, Happy to Help but SIG Testing is not approving this one, so only informally reviewing here.